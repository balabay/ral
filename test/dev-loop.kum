алг loop
нач
	цел a
	ввод a
	нц 20 раз
		a:=a-3
		если a < 10
		то
			вывод "выход", a, нс
			выход
		иначе
			вывод "цикл", a, нс
		всё
		вывод "тело", a, нс
	кц
	вывод "после", a, нс
кон

