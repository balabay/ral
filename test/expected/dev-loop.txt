цикл17
тело17
цикл14
тело14
цикл11
тело11
выход8
